sdl-core/src/main/scala/io/smartdatalake/workflow/dataobject/ExpectationValidation.scala

```diff
@@ -178,7 +180,7 @@ private[smartdatalake] trait ExpectationValidation { this: DataObject with Smart
       })
     // throw exception on error, but log metrics before
     val errors = validationResults.filterKeys(_.failedSeverity == ExpectationSeverity.Error)
-    if (errors.nonEmpty) logger.error(s"($id) Expectation validation failed with metrics "+updatedMetrics.map{case(k,v) => s"$k=$v"}.mkString(" "))
+    if (errors.nonEmpty) logger.error(s"($id) Expectation validation failed with metrics "+orderMetrics(updatedMetrics).map{case(k,v) => s"$k=$v"}.mkString(" "))
```

Comment:
formatting

Reply:
Seems IntelliJ formatting on commit is only concerned about the changed lines.
I reformatted both files now...